src/VirtoCommerce.Platform.Data/ExportImport/PlatformExportImportManager.cs

```diff
@@ -147,137 +147,152 @@ private async Task ImportPlatformEntriesInternalAsync(ZipArchive zipArchive, Pla
             var batchSize = 20;
```

Comment:
I think in case of refactoring you could make it class field - to reduce methods params count